tests/utils/time-units.test.ts

```diff
@@ -2,11 +2,15 @@ import { TimeUnit } from "../../lambdas/common/utils/time-units";
 
 describe("TimeUnit", () => {
   describe("convertTo", () => {
-    it("should return the correct conversion for each unit", () => {
-      expect(TimeUnit.convert(TimeUnit.Unit.Seconds)).toBe(1000);
-      expect(TimeUnit.convert(TimeUnit.Unit.Minutes)).toBe(1000 * 60);
-      expect(TimeUnit.convert(TimeUnit.Unit.Hours)).toBe(1000 * 60 * 60);
-      expect(TimeUnit.convert(TimeUnit.Unit.Days)).toBe(1000 * 60 * 60 * 24);
+    it.each([
+      [TimeUnit.Unit.Seconds, 1000],
+      [TimeUnit.Unit.Minutes, 1000 * 60],
+      [TimeUnit.Unit.Hours, 1000 * 60 * 60],
+      [TimeUnit.Unit.Days, 1000 * 60 * 60 * 24],
+      [TimeUnit.Unit.Months, 1000 * 60 * 60 * 24 * 30],
+      [TimeUnit.Unit.Years, 1000 * 60 * 60 * 24 * 365],
+    ])("should convert %s conversion to %s", (unit, expected) => {
+      expect(TimeUnit.convert(unit)).toBe(expected);
     });
```

Comment:
Nice!! Didn't know you could do this

Reply:
Yeah really nice, ruby does something similar